src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,223 @@
+                insertStatement.setLong(2, driver.getId());
+                insertStatement.executeUpdate();
+            }
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't add drivers dependencies for car with id "
+                    + car.getId(), throwable);
+        }
+    }
+
+    private void removeDrivers(Car car) {
+        String deleteRequest = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement removeStatement = connection.prepareStatement(deleteRequest)) {
+            removeStatement.setLong(1, car.getId());
+            removeStatement.executeUpdate();
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't delete drivers dependencies for car with id "
+                    + car.getId(), throwable);
+        }
+    }
+
+    private List<Driver> getDriversList(Long carId) {
+        String getAllDriversForCarRequest = "SELECT d.id as driver_id,      , "
+                + "d.license_number FROM drivers d "
+                + "JOIN cars_drivers cd ON d.id = cd.driver_id WHERE cd.car_id = ? "
+                + "AND d.is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllDriversStatement
+                         = connection.prepareStatement(getAllDriversForCarRequest)) {
+            getAllDriversStatement.setLong(1, carId);
+            ResultSet resultSet = getAllDriversStatement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                drivers.add(parseDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't get drivers for car with id " + carId,
+                    throwable);
+        }
+    }
+
+    private Car getCarWithManufacturer(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setModel(resultSet.getString("car_model"));
+        car.setId(resultSet.getObject("car_id", Long.class));
+        Manufacturer manufacturer
+                = new Manufacturer(resultSet.getString("manufacturer_name"),
+                resultSet.getString("manufacturer_country"));
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private Driver parseDriver(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver(resultSet.getString("driver_name"),
+                resultSet.getString("licenseNumber"));
+        driver.setId(resultSet.getObject("driver_id", Long.class));
+        return driver;
+    }
+
+    private void driversToListOfCars(List<Car> cars) {
+        cars.forEach(car -> car.setDrivers(getDriversList(car.getId())));
```

Comment:
don't think we need separate method for single line of code

Reply:
done